diff -Nur MT7630E-2.3.4/rt2x00/rt2x00config.c MT7630E-2.3.4n/rt2x00/rt2x00config.c
--- MT7630E-2.3.4/rt2x00/rt2x00config.c	2015-10-01 04:23:43.000000000 +0800
+++ MT7630E-2.3.4n/rt2x00/rt2x00config.c	2015-11-16 16:59:19.521662052 +0800
@@ -270,7 +270,7 @@
 		if (beacon_diff > beacon_int)
 			beacon_diff = 0;
 
-		autowake_timeout = (conf->max_sleep_period * beacon_int) - beacon_diff;
+		autowake_timeout = (conf->ps_dtim_period * beacon_int) - beacon_diff;
 		queue_delayed_work(rt2x00dev->workqueue,
 				   &rt2x00dev->autowakeup_work,
 				   autowake_timeout - 15);
